plugin-core/src/main/java/com/salesforce/bazel/eclipse/builder/BazelMarkerSupport.java

```diff
@@ -77,6 +77,27 @@ protected void execute(IProgressMonitor monitor) throws CoreException {
             }
         });
     }
+    
+    /**
+     * Publishes the specified build errors to the Problems View without clearing it first.
+     *
+     * This method publishes the specified BazelBuildError instances as problem markers to
+     * the Problems View without clearing the problem markers first.
+     *
+     * Note that this method runs within a WorkspaceModifyOperation.
+     *
+     * @param project  the project for which to publish the specified errors
+     * @param errors  the errors to publish to the Problems View
+     * @param monitor  progress monitor
+     */
+    public static void publishToProblemsViewWithoutClearing(IProject project, Collection<BazelBuildError> errors, IProgressMonitor monitor) {
```

Comment:
Instead of adding a "withoutClearing" method, can we just add an explicit clearing method and change the default publish logic to not clear?

Reply:
Will do, that would make it cleaner. I thought about doing that instead but I thought adding a new method would have less impact on existing code. But this is not widely used any way.